Packs/CortexXDR/Playbooks/Cortex_XDR_-_quarantine_file_v2.yml

```diff
@@ -0,0 +1,245 @@
+description: This playbook accepts file path, file hash and endpoint id in order to quarantine a selected file and wait until the action is done.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 698ce1e6-4540-40bb-8ec4-5dc2decebd93
+    type: start
+    task:
+      id: 698ce1e6-4540-40bb-8ec4-5dc2decebd93
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 1809cbad-2546-4991-8fd5-158ef8df6059
+    type: regular
+    task:
+      id: 1809cbad-2546-4991-8fd5-158ef8df6059
+      version: -1
+      name: Quarantine file
+      description: Quarantine a file on selected endpoints. You can select up to 1000 endpoints.
+      script: '|||xdr-file-quarantine'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      endpoint_id_list:
+        complex:
+          root: inputs.endpoint_id
+          transformers:
+          - operator: uniq
+      file_hash:
+        complex:
+          root: inputs.file_hash
+          transformers:
+          - operator: uniq
+      file_path:
+        complex:
+          root: inputs.file_path
+          transformers:
+          - operator: uniq
+      interval_in_seconds:
+        simple: "60"
+      timeout_in_seconds:
+        simple: "600"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 395
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 454d0ef9-46e7-478c-809c-0b0931dce7ce
+    type: title
+    task:
+      id: 454d0ef9-46e7-478c-809c-0b0931dce7ce
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 95f7c67b-1956-4f6c-8455-8549fa1b1d86
+    type: regular
+    task:
+      id: 95f7c67b-1956-4f6c-8455-8549fa1b1d86
+      version: -1
+      name: Get quarantine status
+      description: Retrieves the quarantine status for a selected file.
+      script: '|||xdr-get-quarantine-status'
```

Comment:
after this one, don't we want to check if the operation was successful?
Asking since  this is the last task in the PB